device/thunder_ripple_sdk/src/processors/thunder_device_info.rs

```diff
@@ -952,16 +952,6 @@ impl ThunderDeviceInfoRequestProcessor {
         Err(RippleError::ProcessorError)
     }
 
-    async fn get_timezone(state: CachedState, req: ExtnMessage) -> bool {
-        if let Ok(v) = Self::get_timezone_value(&state).await {
-            Self::respond(state.get_client(), req, ExtnResponse::String(v))
-                .await
-                .is_ok()
-        } else {
-            Self::handle_error(state.get_client(), req, RippleError::ProcessorError).await
-        }
-    }
-
     pub async fn get_timezone_with_offset(state: CachedState, req: ExtnMessage) -> bool {
```

Comment:
Will this function work after the clean up?  This function is used in 

https://github.com/rdkcentral/Ripple/blob/main/core/main/src/service/context_manager.rs#L109